src/lib/components/sidebar/Sidebar.component.js

```diff
@@ -42,6 +44,44 @@ const SidebarContainer = styled.div`
     `;
   }}
 
+  ${(props) => {
+    const { background } = getTheme(props);
+
+    if (props.hoverable && props.hovered && !props.expanded) {
+      return css`
+        .sc-sidebar {
+          position: absolute;
+          height: 100%;
+          background-color: ${background};
+          z-index: 100;
```

Comment:
I suggest we add this to `zIndex` in `theme.json`.
```js
export const zIndex = {
  tooltip: 9990,
  notification: 9000,
  modal: 8500,
  overlay: 8000,
  dropdown: 7000,
  nav: 500,
  sidebar: 100,
  base: 1,
};
```

Reply:
Done :)